Changelog.md

```diff
@@ -1,6 +1,10 @@
-### Development
+# Development
 [Full Changelog](http://github.com/rspec/rspec-expectations/compare/v3.10.0...main)
 
+## Breaking Changes
```

Comment:
```suggestion
Breaking Changes:
```

Reply:
Sorry I didn't mean to break consistency with my first suggestion

Reply:
No worries!